cla_frontend/settings/docker.py

```diff
@@ -1,47 +1,13 @@
 from .base import *
 import os
 
-try:
-    SECRET_KEY = os.environ["SECRET_KEY"]
```

Comment:
base.py still has SECRET_KEY hard-coded to a string. I think we need to move getting it from an env var up into that file